src/station_cmd.cpp

```diff
@@ -3794,6 +3794,8 @@ void IncreaseStats(Station *st, CargoID cargo, StationID next_station_id, uint c
  */
 void IncreaseStats(Station *st, const Vehicle *front, StationID next_station_id)
 {
+	bool current_order_is_implicit = front->current_order.IsType(OT_IMPLICIT);
```

Comment:
Testing for "implicit" orders looks wrong to me. Doesn't the same happen for explicit orders?
I would expect testing for "not forced unload/transfer".